src/app/(pages)/my-page/coupon-page/components/CouponContents.tsx

```
@@ -0,0 +1,74 @@
+      <div className="flex mt-[15px] mx-auto w-[95%] justify-between items-center md:items-center md:justify-center md:gap-4">
+        <div>
+          <button
+            className={`px-6 py-2 border-b-4 text-[16px] ${
+              activeTab === 'coupons'
+                ? 'text-primary-20 border-primary-20'
+                : 'text-label-assistive border-transparent'
+            }`}
+            onClick={() => setActiveTab('coupons')}
+          >
+            사용 가능 쿠폰
+          </button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)